app/app/api/public-cloud/products/[licencePlate]/billings/[billingId]/sign/route.ts

```diff
@@ -72,6 +71,24 @@ export const POST = apiHandler(async ({ pathParams, body, session }) => {
   });
 
   if (billingDecorated) {
+    const existingReviewTask = await prisma.task.findFirst({
```

Comment:
You don't need to check whether it exists or not, just use `updateMany` to cancel them all even though there will be just one record matched